docs/installation/installation-using-docker-on-production-server.md

```diff
@@ -1,7 +1,10 @@
 # Installation Using Docker on Production Server
 
-This guide shows how to install and configure production server applications needed for running [shopsys/project-base](https://github.com/shopsys/project-base).
-We use docker containers for `php` applications and natively installed applications like `postgres`, `elasticsearch`, `redis` for persisting data without loosing them after `project-base` deployments.
+This guide shows you how to install and configure production server applications needed for your project based on [Shopsys Framework](https://github.com/shopsys/project-base).  
+We use natively installed applications like postgres, elasticsearch, redis for persisting data without loosing them after deploy of new version of the project.
+We do not want to setup each application or each micro-service application manually and in server environment and have some separation between them.
```

Comment:
I would use this sentence right after the first one ("This guide shows you how to install and configure production server applications needed for your project based on Shopsys Framework").

Reply:
also, ~~micro-service~~ -> microservice

Reply:
also, the part "...and in server environment and have some separation between them." is unclear to me.